client/my-sites/comments/comment-list/index.jsx

```diff
@@ -176,14 +151,13 @@ export class CommentList extends Component {
 				>
 					{ map( filteredComments, comment =>
 						<CommentDetail
-							commentId={ comment.ID }
-							deleteForever={ this.deleteForever( comment.ID ) }
+							comment={ comment }
+							deleteCommentPermanently={ this.deleteCommentPermanently( comment.ID ) }
```

Comment:
Oh I meant that the child component could pass the commentId, since it already has it available. Very roughly this might look like:

In CommentList:
```jsx
deleteCommentPermanently={ this.deleteCommentPermanently }
```

And in the child component CommentDetail:
```jsx
deleteCommentPermanently = () => {
    this.props.deleteCommentPermanently( this.props.commentId );
}
```